frontend/src/Pages/LoginPage/LoginPage.jsx

```diff
@@ -11,9 +11,18 @@ export const LoginPage = () => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [error, setError] = useState("")
-  const [userData, setUserData] = useState("")
-  const [token, setToken] = useState("")
+  const [userData, setUserData] = useState("");
+  const token = localStorage.getItem("token");
   const navigate = useNavigate();
+  
+  // get token from local storage
+  
+  useEffect(() => {
```

Comment:
@annagwozdz - I've added a useEffect to check for the presence of the token, Navigate back to 'animals' if not